jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/upload/LambdaLineMarker.kt

```diff
@@ -64,6 +72,11 @@ class LambdaLineMarker : LineMarkerProviderDescriptor() {
     ) {
     }
 
+    private fun isHandlerValid(element: PsiElement, handler: String, runtime: Runtime): Boolean =
```

Comment:
It's not really `isValid` is more like `shouldShowLineMarker`

Reply:
Renamed.